docs/blog/20230903-confused-Techie-pulsars-ci.md

```diff
@@ -0,0 +1,141 @@
+
+| Credit-Cost Build Type | Credits | Builds/month |
+| ---------------------- | ------- | ------------ |
+| Lowest Recorded        | 1.032   | 48           |
+| Median                 | 1.256   | 39           |
+| Average                | 1.269   | 39           |
+| Highest Recorded       | 1.766   | 28           |
+
+With these exact numbers of our current trends, we could then assume we have an allotment of 28 builds per month for these platforms. But consdering the possible need of retries, or a build erroring out, we had settled on building Apple Silicon, and ARM Linux once every other day. Which should give us about 15 builds per month minimum. This would give us a healthy buffer zone, or any retries needed, and the ability to do another run for the regular release every month.
+
+But even though we now knew what we needed to do, there was still a long list of things we had to change to get there, especially now considering our deadline to do so was only a few days away.
+
+After a flurry of PR's and lots of testing, the team at Pulsar was able to accomplish all of the following in just under our 5 day limit:
+
+- Migrated all other platforms over to GitHub Actions
+- Ensure binaries are easily downloadable via GitHub Actions for testing purposes
+- Configure CRON jobs within Cirrus to build these platforms every two days
+- Minimize the amount of times Cirrus would run, ensuring to only run if changes were present, and disallow any other type of run that may occur
+- Setup a brand new repository `pulsar-rolling-releases` to act as the CDN for every single rolling release
+- Nearly completely rewrite the `download` microservice to now use this new repository to retreive the newest rolling release
+
+All of that work has culminated in a new CI plan for Pulsar, the one that is currently in effect starting September. On every single PR to the Pulsar repo, a binary will be built for Windows, Linux, and macOS. Once that PR is merged, a new rolling release will automatically be built, again just for these three platforms, which will then be publishing as a new release to the repository [`pulsar-rolling-releases`](https://github.com/pulsar-edit/pulsar-rolling-releases). Once on this repository the `download` microservice will be able to find and return these versions to any users attempting to download a rolling release of Pulsar. Then once every two days, Cirrus will be triggered to build binaries of Apple Silicon and ARM Linux. These new binaries will also be published to `pulsar-rolling-releases`. Then whenever we plan to do a new regular release of Pulsar, once a new tag is created for this release, Cirrus will automatically be triggered to build binaires for it's two platforms. Luckily for us, all of these changes mean we have not incurred any additional cost for the Pulsar project, even though there is some talk of officially supporting Cirrus on Open Collective at a price we can properly afford.
```

Comment:
```suggestion
All of that work has culminated in a new CI plan for Pulsar, the one that is currently in effect starting September. On every single PR to the Pulsar repo, a binary will be built for Windows, Linux, and macOS. Once that PR is merged, a new rolling release will automatically be built, again just for these three platforms, which will then be publishing as a new release to the repository [pulsar-rolling-releases](https://github.com/pulsar-edit/pulsar-rolling-releases). Once on this repository the `download` microservice will be able to find and return these versions to any users attempting to download a rolling release of Pulsar. Then once every two days, Cirrus will be triggered to build binaries of Apple Silicon and ARM Linux. These new binaries will also be published to `pulsar-rolling-releases`. Then whenever we plan to do a new regular release of Pulsar, once a new tag is created for this release, Cirrus will automatically be triggered to build binaires for its two platforms. Luckily for us, all of these changes mean we have not incurred any additional cost for the Pulsar project, even though there is some talk of officially supporting Cirrus on Open Collective at a price we can properly afford.
```

Reply:
At least for me, GH is failing to display the diff here. The suggestion removes the \` inline code blocks from inside the pulsar-rolling-releases hyperlink as I don't think it is needed there are it already displays differently due to the link itself.

I also removed a \' from "It's"